README.md

```diff
@@ -44,139 +70,134 @@ source /usr/share/virtualenvwrapper/virtualenvwrapper.sh
 
 ### Enable I2C
 
+Use the Raspberry Pi configuration tool to enable I2C:
 `sudo raspi-config` -> `Interface Options` -> `I2C`
 
-TODO set 400KHz ?
-
-### Set the usbserial latency timer
-
-```bash
-cd  /etc/udev/rules.d/
-sudo touch 99-usb-serial.rules
-sudo nano 99-usb-serial.rules
-# copy the following line in the file
-SUBSYSTEM=="usb-serial", DRIVER=="ftdi_sio", ATTR{latency_timer}="1"
-```
-
-### Set the udev rules for the motor control board
+*(TODO: Set to 400KHz?)*
 
-TODO
-
-
-### Setup xbox one controller over bluetooth
-
-Turn your xbox one controller on and set it in pairing mode by long pressing the sync button on the top of the controller.
-
-Run the following commands on the rasp :
+### Set the USB Serial Latency Timer
 
+Create a udev rule to set the latency timer for the USB-to-serial adapter:
 ```bash
-bluetoothctl
-scan on
+sudo nano /etc/udev/rules.d/99-usb-serial.rules
 ```
-
-Wait for the controller to appear in the list, then run :
-
-```bash
-pair <controller_mac_address>
-trust <controller_mac_address>
-connect <controller_mac_address>
+Add the following line to the file:
 ```
-
-The led on the controller should stop blinking and stay on.
-
-You can test that it's working by running
-
-```bash
-python3 mini_bdx_runtime/mini_bdx_runtime/xbox_controller.py
+SUBSYSTEM=="usb-serial", DRIVER=="ftdi_sio", ATTR{latency_timer}="1"
 ```
 
-## Speaker wiring and configuration
-Follow this tutorial
-
-> For now, don't activate `/dev/zero` when they ask
-
-https://learn.adafruit.com/adafruit-max98357-i2s-class-d-mono-amp?view=all
+### Motor Control Board udev Rules
 
+*(TODO)*
 
-## Install the runtime
+## Install the Runtime
 
-### Make a virtual environment and activate it
+### Make a Virtual Environment and Activate it
 
 ```bash
 mkvirtualenv -p python3 open-duck-mini-runtime
 workon open-duck-mini-runtime
 ```
 
-Clone this repository on your rasp, cd into the repo, then :
+### Install the Repository
 
+Clone the repository and install it in editable mode:
 ```bash
 git clone https://github.com/apirrone/Open_Duck_Mini_Runtime
 cd Open_Duck_Mini_Runtime
-git checkout v2
 pip install -e .
 ```
 
-In Raspberry Pi 5, you need to perform the following operations
-
+**For Raspberry Pi 5:** You may need to replace the GPIO library.
```

Comment:
This shouldn't be necessary since https://github.com/apirrone/Open_Duck_Mini_Runtime/pull/23 has been merged. We use the same GPIO library for pi zero and pi5